build_files/20-install-apps.sh

```diff
@@ -3,11 +3,29 @@ set -xeuo pipefail
 
 dnf5 install -y \
     android-tools \
+    bcc \
+    bpftop \
+    bpftrace \
+    flatpak-builder \
+    nicstat \
+    numactl \
     openrgb \
+    p7zip-plugins \
+	p7zip \
```

Comment:
```suggestion
    p7zip \
```
Formatting is gonna complain